### DISCUSSION TITLE
how to add colab another model?

### DISCUSSION THREAD

**Discussion by rafiislambd** [lllyasviel/Fooocus#864] [General]
how to add colab another model?

**Comment by botzero-net** [comment]
Add the wget command to the colab including the model source and destination paths.  Here are some examples.

!wget -c https://civitai.com/api/download/models/130072 -O ./models/checkpoints/realisticVisionV51_v51VAE.ckpt
!wget -c https://civitai.com/api/download/models/147053 -O ./models/loras/taco-dog.safetensors
!wget -c https://huggingface.co/AIARTCHAN/aichan_blend/resolve/main/vae/BerrysMix.vae.safetensors -P ./models/vae/

Note the different formatting needed for Civit vs. Huggingface, and the exclamation point at the beginning is needed for the notebook to process the command.  You can also simply click the folder icon on the left side of the colab screen, expand out to the target folder, and click the ... to upload a file manually, but you'll have to do this every session.  Adding the commands above and saving the notebook allows you to have the new models every time you run it again.  Hope this helps!

**Comment by KitsonBroadhurst** [comment]
Quick video here on it: https://www.youtube.com/watch?v=E37Tdcx2DyA
